build.sh

```diff
@@ -128,18 +128,42 @@ set_status() {
     /usr/bin/curl -s -d "${status}" -H "Content-Type: application/json" -H "Authorization: ${CI_JOB_TOKEN}" -X PUT ${MURDOCK_API_URL}/job/${CI_JOB_UID}/status > /dev/null
 }
 
+check_label() {
+    local label="${1}"
+    [ -z "${CI_PULL_LABELS}" ] && return 1
+    echo "${CI_PULL_LABELS}" | grep -q "${label}"
+    return $?
+}
+
+if [ -z "${CI_FAST_FAIL}" ]; then
+    if [ "${NIGHTLY}" = "1" ]; then
+        export CI_FAST_FAIL=0
+    elif check_label "CI: no fast fail"; then
+        export CI_FAST_FAIL=0
+    else
+        export CI_FAST_FAIL=1
+    fi
+fi
+
+if [ "${CI_FAST_FAIL}" = "1" ]; then
+    export DWQ_MAXFAIL=0
+else
+    export DWQ_MAXFAIL=500
+fi
+
 main() {
     set_status "fetching code"
 
-    export APPS BOARDS
+    export APPS BOARDS NIGHTLY STATIC_TESTS RUN_TESTS
+
+    export DWQ_ENV="-E APPS -E BOARDS -E NIGHTLY -E STATIC_TESTS -E RUN_TESTS \
+                    -E CI_MURDOCK_PROJECT -E ENABLE_TEST_CACHE -E CI_FAST_FAIL \
+                    -E FULL_BUILD"
 
     local repo_dir="RIOT"
     if [ -n "${CI_BUILD_COMMIT}" ]; then
-        export NIGHTLY STATIC_TESTS
```

Comment:
NIGHTLY and STATIC_TESTS envs only make sense when building a commit/branch/tag. This is why they are only exported there.
What is the reason for exporting them globally ?

Reply:
(STATIC_TESTS does make sense for PRs IIUC)